zetaclient/chains/evm/observer/inbound.go

```diff
@@ -288,12 +280,12 @@ func (ob *Observer) ObserveZetaSent(ctx context.Context, startBlock, toBlock uin
 
 // ObserveERC20Deposited queries the ERC20CustodyDeposited event from the ERC20Custody contract and posts to zetacore
 // returns the last block successfully scanned
-func (ob *Observer) ObserveERC20Deposited(ctx context.Context, startBlock, toBlock uint64) uint64 {
+func (ob *Observer) ObserveERC20Deposited(ctx context.Context, startBlock, toBlock uint64) (uint64, error) {
 	// filter ERC20CustodyDeposited logs
 	addrCustody, erc20custodyContract, err := ob.GetERC20CustodyContract()
 	if err != nil {
 		ob.Logger().Inbound.Warn().Err(err).Msgf("ObserveERC20Deposited: GetERC20CustodyContract error:")
```

Comment:
Do we still need the logs here if we return the error? Can be logged in the calling function

Reply:
Removed log prints in `ObserveZetaSent, ObserveERC20Deposited, ObserveTSSReceive` https://github.com/zeta-chain/node/pull/3496/commits/25a66b204f29f8e0d3d1a47bc1e7661635331b1b